server.js

```diff
@@ -622,6 +622,20 @@ app.use('/pl/course_instance/:course_instance_id/instance_question/:instance_que
     require('./pages/clientFilesQuestion/clientFilesQuestion'),
 ]);
 
+// questionThumbnail
+app.use('/pl/course/:course_id/course_admin/question_thumbnail', [
+    function(req, res, next) {next();},
+    require('./pages/questionThumbnail/questionThumbnail'),
+]);
+app.use('/pl/course_instance/:course_instance_id/instructor/course_admin/question_thumbnail', [
+    function(req, res, next) {next();},
+    require('./pages/questionThumbnail/questionThumbnail'),
+]);
+app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/question_thumbnail', [
+    function(req, res, next) {next();},
+    require('./pages/questionThumbnail/questionThumbnail'),
+]);
+
```

Comment:
Can we just use a single route of `/pl/course/:course_id/question/:question_id/thumbnail`? I don't think we need to have three different routes here.